lib/fog/proxmox/compute/models/servers.rb

```diff
@@ -53,10 +53,16 @@ def get(id)
             status_data = service.get_server_status path_params
             config_data = service.get_server_config path_params
           rescue StandardError => e
-            if e.respond_to?('response') && e.response.respond_to?('data') && e.response.data.has_key?(:reason_phrase) && e.response.data[:reason_phrase].end_with?('does not exist')
-              raise(Fog::Errors::NotFound)
+            if e.respond_to?('response') && e.response.respond_to?('data') && e.response.data.has_key?(:body)
```

Comment:
handling body additionally, might be useful - but we still should catch the reason_pharse, too?